<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtres2doxClass</class>
 <widget class="QMainWindow" name="qtres2doxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtres2dox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSelectResource"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Image"/>
    <addaction name="actionDoxygenExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>OpenForm  File (*.ui)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport_Image">
   <property name="text">
    <string>Export Form as Image</string>
   </property>
  </action>
  <action name="actionSelectResource">
   <property name="text">
    <string>Select Resource (*.qrc)</string>
   </property>
  </action>
  <action name="actionDoxygenExport">
   <property name="text">
    <string>Doxygen Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtres2dox.qrc"/>
 </resources>
 <connections/>
</ui>
